python/ray/experimental/workflow/api.py

```diff
@@ -42,38 +40,22 @@ def run(entry_workflow: Workflow,
             checkpointing.
         workflow_id: The ID of the workflow. The ID is used to identify
             the workflow.
+        detached: If True, the execution of workflow is detached form
```

Comment:
```suggestion
        detached: If True, the execution of the workflow is decoupled from the current
            process. Otherwise, it will fate share with the current driver, actor, or task.
```